server/app.js

```diff
@@ -1,39 +1,15 @@
-const createError = require("http-errors");
-const express = require("express");
-const { join } = require("path");
-const cookieParser = require("cookie-parser");
-const logger = require("morgan");
+import express from "express";
+import dotenv from "dotenv";
+import connectDB from "./config/db.js";
+import { join } from "path"
+import morgan from "morgan";
 
-const indexRouter = require("./routes/index");
-const pingRouter = require("./routes/ping");
+dotenv.config()
 
-const { json, urlencoded } = express;
+connectDB()
```

Comment:
nitpick: inconsistency on presence of semi colons - i'd recommend to have it set up